test/sw_mgmt_spec.rb

```diff
@@ -294,15 +294,15 @@
       subject.copy_old_credentials(root_dir)
     end
 
-    it "copies old SMT credentials at upgrade" do
-      smt_credentials = File.join(root_dir, target_dir, "SMT-http_smt_example_com")
+    it "copies old SMT/RMT credentials at upgrade" do
+      credentials = File.join(root_dir, target_dir, "SMT-RMT-http_example_com")
```

Comment:
NP: probably it should be just `RMT-http_example_com`, but it actually does not matter as YaST copied everything regardless the name.